docs/application/native/guides/connectivity/overview.md

```diff
@@ -18,10 +18,14 @@ You can use smart card functionalities to access secure elements. You can open a
 You can connect to a Wireless Local Area Network (WLAN) and transfer data over the network. You can activate and deactivate a local Wi-Fi device, and to connect to a WLAN network in the infrastructure mode.
 
 - [Wi-Fi Direct&reg;](wifi-direct-n.md) **in mobile applications only**
-You can find nearby Wi-Fi Direct devices and discover their details. You can also form a Wi-Fi Direct group to communicate over a peer-to-peer link without wireless access points (base stations) in the infrastructure mode.
+Find Wi-Fi Direct devices, discover their details, and form Wi-Fi Direct group to communicate over a peer-to-peer link without wireless access points (base stations) in the infrastructure mode.
+
+- [STC (Smart Traffic Control)](stc.md)
+Statistics about network data consumption is fetched by system and applications.
 
 - [Telephony Information](telephony.md)
-You can access various information from the Telephony Service. You can access details about the voice and video call states, currently used SIM card, and modem. You can also retrieve general details about the telephony network the device is currently connected to.
+1. Access various information about voice and voice call status that currently uses SIM card and Modem from the Telephony Service.
+2. Retrieving general details about the telephony network from the currently connected device.
```

Comment:
Sentence re-write is correct. No further changes needed.

Reply:
done